src/Authorization/Authorization.router.js

```diff
@@ -0,0 +1,118 @@
+import { server, env } from 'decentraland-commons'
+
+import { Approval } from '../Approval'
+import { unsafeParseInt } from '../lib'
+import { Parcel, Estate } from '../Asset'
+import { ASSET_TYPES } from '../shared/asset'
+
+export class AuthorizationRouter {
+  constructor(app) {
+    this.app = app
+  }
+
+  mount() {
+    /**
+     * Returns the bids for a parcel
+     * @param  {string} x
+     * @param  {string} y
+     * @param  {string} [address] - address to get authorizations
+     * @return {<id, address, isOwner, isApprovedForAll, isOwner, isOperator, isUpdateOperator>}
+     */
+    this.app.get(
+      '/parcels/:x/:y/:address/authorizations',
+      server.handleRequest(this.getParcelAuthorizations.bind(this))
+    )
+
+    /**
+     * Returns the bids for a parcel
+     * @param  {string} id
+     * @param  {string} [address] - address to get authorizations
+     * @return {<id, address, isOwner, isApprovedForAll, isOwner, isOperator, isUpdateOperator>}
+     */
+    this.app.get(
+      '/estates/:id/:address/authorizations',
+      server.handleRequest(this.getEstateAuthorizations.bind(this))
+    )
+  }
+
+  async getAuthorizations(asset, assetType, address) {
+    let tokenAddress
+
+    switch (assetType) {
+      case ASSET_TYPES.parcel: {
+        tokenAddress = env.get('LAND_REGISTRY_CONTRACT_ADDRESS')
+        break
+      }
+      case ASSET_TYPES.estate: {
+        tokenAddress = env.get('ESTATE_REGISTRY_CONTRACT_ADDRESS')
+        break
+      }
+      default:
+        throw new Error(`The assetType ${assetType} is invalid`)
+    }
+
+    const isApprovedForAll = await Approval.isApprovedForAll(
+      tokenAddress.toLowerCase(),
+      asset.owner,
+      address.toLowerCase()
+    )
+
+    return {
+      isApprovedForAll,
+      isOwner: asset.owner === address,
+      isOperator: asset.operator === address,
+      isUpdateOperator: asset.update_operator === address
+    }
+  }
+
+  async getParcelAuthorizations(req) {
+    let x, y
+
+    try {
+      x = unsafeParseInt(server.extractFromReq(req, 'x'))
+    } catch (e) {
+      throw new Error('Invalid coordinate "x" must be an integer')
+    }
+
+    try {
+      y = unsafeParseInt(server.extractFromReq(req, 'y'))
+    } catch (e) {
+      throw new Error('Invalid coordinate "y" must be an integer')
+    }
```

Comment:
We should use `ReqQueryParams` here which abstracts `unsafeParseInt` in `getInteger`

`const x = reqQueryParams.getInteger('x')`